crates/cdk-sql-common/src/common.rs

```diff
@@ -0,0 +1,49 @@
+use crate::database::DatabaseExecutor;
+use crate::stmt::query;
+
+/// Migrates the migration generated by `build.rs`
+#[inline(always)]
+pub async fn migrate<C: DatabaseExecutor>(
+    conn: &C,
+    db_prefix: &str,
+    migrations: &[(&str, &str)],
+) -> Result<(), cdk_common::database::Error> {
+    query(
+        r#"
+           CREATE TABLE IF NOT EXISTS migrations (
+               name TEXT PRIMARY KEY,
+               applied_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+           )
+           "#,
+    )?
+    .execute(conn)
+    .await?;
+
+    // Apply each migration if it hasn’t been applied yet
+    for (name, sql) in migrations {
+        let basename = if let Some((prefix, basename)) = name.split_once(['/', '\\']) {
+            if prefix != db_prefix {
+                continue;
+            }
+            basename
+        } else {
+            name
+        };
```

Comment:
```suggestion
let basename = match name.split_once(['/', '\\']) {
    Some((prefix, basename)) => {
        if prefix != db_prefix {
            continue;
        }
        basename
    }
    None => name,
};
```

Our code style says we should use match here https://github.com/cashubtc/cdk/blob/main/CODE_STYLE.md#if-let.